pkg/ccl/backupccl/backup_processor.go

```diff
@@ -82,6 +84,13 @@ var (
 		"split backup data on timestamps when writing revision history",
 		true,
 	)
+
+	sendExportRequestWithVerboseTracing = settings.RegisterBoolSetting(
+		settings.TenantWritable,
+		"bulkio.backup.export_request_verbose_tracing",
```

Comment:
How come this name was validated by the test `TestLintClusterSettingNames` in `pkg/sql/show_test.go`? It's not valid per that linter. Ditto the `_delay` one above.